### ISSUE TITLE
Getting error: command deepscan.inspectProject not found

### ISSUE THREAD

**Issue by JaiMistry** [deepscan/vscode-deepscan#24]
After installing and restarting VScode, I didn't receive a prompt for a Confirm button like the documentation said it would. I manually went into VScode settings and enabled: `Deepscan: Enable`
However, I don't see analysis happening. I also tried to use `ctrl+shit+p` to manually execute `DeepScan: Inspect Project` and that's when I receive the error mentioned in the title.

VS Code Version: 1.43.1 (user setup)

**Comment by cimfalab** [comment]
Maybe the extension seems not to be installed properly.
While "Inspect Project" command is only for the embedded mode, the command should be executed anyway if the extension was installed.

Try the following:
1. Uninstall our extension
2. Install our extension
3. Check whether you can see our extension in the Extensions activity
4. Check logs in the Output panel with "Log (Extension Host)"

FYI: If you installed the extension and confirmed, you can see `"deepscan.enable": true` in the settings.json of a user or workspace.

**Comment by JaiMistry** [comment]
It is working now. I created a new folder and created a `.js` inside of it to test. So, it seems that the extension does not work on files that are not contained within a folder?

**Comment by cimfalab** [comment]
You got a point.
After I close a folder and open a file directly, the extension is not activated.
Maybe I think it's because there is no workspace root.
```
export function activate(context: vscode.ExtensionContext) {
    const workspaceRootPath = vscode.workspace.rootPath;
    if (!workspaceRootPath) {
        return;
    }
```
I will have a further look after the weekend.
Thanks.

**Comment by cimfalab** [comment]
I just published a v1.9.7 extension.
It's changed to continue an activation although if VS Code is not opened on a workspace folder.
Then when you set `deepscan.enable` to `true` in the user settings, it will inspect an opened file.
As of the command, it is registered (not 'not found' anymore) and can display an error message for the proper case.
Thank you for the issue.
